app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
@ShadyShawkat The project requirement says we should ensure **Hashed password should be stored in the database.** If i go by the exact wireframe, then i wont be able to do this because the remember me will be removed

Reply:
@ShadyShawkat The project requirement also said we should be able to **reset password**, which is where the **forgot password** falls in.

See project requirement below;
![image](https://user-images.githubusercontent.com/97225349/186693405-1037bd92-abb6-4561-b8ad-13a0864fa34f.png)

Reply:
@ShadyShawkat Also advise on this one

Reply:
Hi @Adanzeakonobi, you can still use those link at the bottom but the style has to be at least close to the given wireframe. I'm talking about the inputs, header, and "Login" button.